.make/test.mk

```diff
@@ -117,6 +117,11 @@ ALL_PKGS_EXCLUDE_PATTERN = 'vendor\|app\|tool\/cli\|design\|client\|test'
 GOANALYSIS_PKGS_EXCLUDE_PATTERN="vendor|app|client|tool/cli"
 GOANALYSIS_DIRS=$(shell go list -f {{.Dir}} ./... | grep -v -E $(GOANALYSIS_PKGS_EXCLUDE_PATTERN))
 
+MINISHIFT_USER_NAME ?= developer$(shell date +'%H.%M.%S')
+MINISHIFT_USER_TOKEN ?= $(shell oc login -u=$(MINISHIFT_USER_NAME) -p=developer > /dev/null && oc whoami -t)
+MINISHIFT_ADMIN_NAME ?= admin
```

Comment:
I don't create a new `admin` because it is not needed - the admin is used only for having `cluster-admin` rights when creating/updating/deleting the namespaces.

Reply:
thanks for explaination